### PR TITLE
cpupower.py -> Providing fix to install packages

### PR DESCRIPTION
**Pull request by shriyak**
Fix to install pakcages to support cpupower utility.Ubuntu supports linux tools package and other distro supports
powerpc-utils.Also provided fix to change scehdutils->schedultil.

Signed-off-by: Shriya <shriyak@linux.vnet.ibm.com>

### CODE REVIEW

File: cpu/cpupower.py [hunk 1]
```diff
@@ -15,16 +15,31 @@
...
+    def setUp(self):
+        smm = SoftwareManager()
+        detected_distro = distro.detect()
+        kernel_ver = platform.uname()[2]
+        if 'Ubuntu' in detected_distro.name:
+            deps = ['linux-tools-common', 'linux-tools-%s' % kernel_ver]
+        else:
+            deps = ['powerpc-utils']
+
+        #self.log.info("Install the package to run cpupower tool")
```
Comment (narasimhan-v) [inline cpu/cpupower.py]: Please remove this

File: cpu/cpupower.py [hunk 2]
```diff
@@ -119,7 +137,8 @@ def set_freq_val(self, freq):
         output = process.run(cmd)
         cur_freq = self.get_cur_freq()
         if (output.exit_status == 0) and (cur_freq == freq):
-            self.log.info("the userspace governor working as expected and freeq set correctly")
+            self.log.info("the userspace governor working
+                          as expected and freeq set correctly")
```
Comment (narasimhan-v) [inline cpu/cpupower.py]: freq

Comment (shriyak) [inline cpu/cpupower.py]: added the changes

**Comment by shriyak** [comment]
Raised new PR

**Pull request closed without merging**